### PR TITLE
Fix logging formatting

### PR DESCRIPTION
**Pull request by vladmonea**
Formatting in MinionError references a a field 3, which should be a field 2.  May cause an exception if executed.

Note: the change-set is quite big due to linter requirements. The actual change is on line 691. Additionally, I've removed unused arguments on lines 44 and 260, respectively.

### CODE REVIEW

File: hubblestack/fileclient.py [hunk 1]
```diff
@@ -492,17 +467,19 @@ def get_url(self, url, dest, makedirs=False, saltenv='base',
...
             if source_hash is not None:
                 try:
-                    source_hash = source_hash.split('=')[-1]
+                    source_hash = source_hash.split("=")[-1]
                     form = hubblestack.utils.files.HASHES_REVMAP[len(source_hash)]
                     if hubblestack.utils.hashutils.get_hash(dest, form) == source_hash:
                         log.debug(
-                            'Cached copy of %s (%s) matches source_hash %s, '
-                            'skipping download', url, dest, source_hash
+                            "Cached copy of %s (%s) matches source_hash %s, " "skipping download",
```
Comment (grogsaxle) [inline hubblestack/fileclient.py:479]: Unnecessary " " in log.debug() between %s, and skipping
`"Cached copy of %s (%s) matches source_hash %s, skipping download",`

File: hubblestack/fileclient.py [hunk 2]
```diff
@@ -813,23 +775,15 @@ def get_file(self,
         if not dest2check:
             rel_path = self._check_proto(path)
 
-            log.debug(
-                'In saltenv \'%s\', looking at rel_path \'%s\' to resolve '
-                '\'%s\'', saltenv, rel_path, path
-            )
-            with self._cache_loc(
-                    rel_path, saltenv, cachedir=cachedir) as cache_dest:
+            log.debug("In saltenv '%s', looking at rel_path '%s' to resolve " "'%s'", saltenv, rel_path, path)
```
Comment (grogsaxle) [inline hubblestack/fileclient.py:778]: Unecessary " " in middle og log.debug.   Maybe change to:
`log.debug("In saltenv '%s', looking at rel_path '%s' to resolve '%s'", saltenv, rel_path, path)

File: hubblestack/fileclient.py [hunk 3]
```diff
@@ -813,23 +775,15 @@ def get_file(self,
...
+            with self._cache_loc(rel_path, saltenv, cachedir=cachedir) as cache_dest:
                 dest2check = cache_dest
 
-        log.debug(
-            'In saltenv \'%s\', ** considering ** path \'%s\' to resolve '
-            '\'%s\'', saltenv, dest2check, path
-        )
+        log.debug("In saltenv '%s', ** considering ** path '%s' to resolve " "'%s'", saltenv, dest2check, path)
```
Comment (grogsaxle) [inline hubblestack/fileclient.py:782]: Unnecessary " " in middle of log.debug.  Maybe change to: 
`log.debug("In saltenv '%s', ** considering ** path '%s' to resolve '%s'", saltenv, dest2check, path)`

**Review by grogsaxle — CHANGES_REQUESTED** [review]
@vladmonea Nice cleanup.   Couple of places to tweak on log.debug() and then will be good to go.

**Review by mzainea — APPROVED** [review]

**Pull request merged**